MapboxGLAndroidSDK/src/cpp/style/sources/source.cpp

```diff
@@ -180,6 +180,23 @@ static std::unique_ptr<Source> createSourcePeer(jni::JNIEnv& env,
         return ownedSource != nullptr;
     }
 
+    jni::Local<jni::Boolean> Source::isVolatile(jni::JNIEnv &env) {
+        return jni::Box(env, jni::jboolean(source.isVolatile()));
+    }
+
+    void Source::setVolatile(JNIEnv &env, jni::Boolean & value) {
+        source.setVolatile(jni::Unbox(env, value));
+    }
+
+    void Source::setMinimumTileUpdateInterval(JNIEnv& env, jni::Long & interval) {
+        jlong i = jni::Unbox(env, interval);
```

Comment:
1 letter name doesn't say much about the object, either improve name or fold into below.